nailgun/entity_mixins.py

```diff
@@ -1245,6 +1261,9 @@ def search(self, fields=None, query=None, filters=None):
         ]
         if filters is not None:
             entities = self.search_filter(entities, filters)
+
+        signals.post_search.send(self, entities=entities, fields=fields,
+                                 query=query, filters=filters)
```

Comment:
Do you think of any scenario in which signals will be useful for a search operation?